Lib/test/test_smtplib.py

```diff
@@ -1227,7 +1230,103 @@ def test_send_message_error_on_non_ascii_addrs_if_no_smtputf8(self):
             HOST, self.port, local_hostname='localhost', timeout=3)
         self.addCleanup(smtp.close)
         self.assertRaises(smtplib.SMTPNotSupportedError,
-                          smtp.send_message(msg))
+                          lambda: smtp.send_message(msg))
```

Comment:
This assertRaises call should read `self.assertRaises(smtplib.SMTPNotSupportedError, smtp.send_message, msg)`.  The fact that this test isn't failing as-is means it isn't being run, which is issue #32663, and we should fix that one before applying your patch.

Reply:
After I changed this. I saw you merged a fix for [bpo-32663](https://bugs.python.org/issue32663). I will merge.